x-pack/plugins/cases/public/components/case_view/components/edit_catecory.test.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { EditCategoryProps } from './edit_category';
+import { EditCategory } from './edit_category';
+import { readCasesPermissions } from '../../../common/mock';
+import userEvent from '@testing-library/user-event';
+import type { AppMockRenderer } from '../../../common/mock';
+import { createAppMockRenderer } from '../../../common/mock';
+
+import { waitFor, screen } from '@testing-library/react';
+import { useGetCategories } from '../../../containers/use_get_categories';
+
+jest.mock('../../../containers/use_get_categories');
+
+const onSubmit = jest.fn();
+const defaultProps: EditCategoryProps = {
+  isLoading: false,
+  onSubmit,
+  category: '',
+};
+
+describe('EditCategory ', () => {
+  let appMockRender: AppMockRenderer;
+  const sampleCategories = ['foo', 'bar'];
+  const fetchCategories = jest.fn();
+  // const formHookMock = getFormMock({ categories: sampleCategories });
```

Comment:
Can we remove this commented code? (here and line 37)

Reply:
I think you are looking an outdated code